core/src/main/java/org/infinispan/util/logging/Log.java

```diff
@@ -542,8 +542,8 @@ void preparedTxAlreadyExists(RecoveryAwareTransaction previous,
    void unableToLockToInvalidate(Object key, Address address);
 
    @LogMessage(level = ERROR)
-   @Message(value = "Execution error", id = 136)
-   void executionError(@Cause Throwable t);
+   @Message(value = "Error executing command %s, writing keys %s", id = 136)
+   void executionError(String commandType, Collection<Object> affectedKeys, @Cause Throwable t);
```

Comment:
This seems like a lot of trace possibly to go to ERROR.  Maybe we should only write all the keys if TRACE is enabled?